static/js/settings_invites.js

```diff
@@ -16,10 +16,10 @@ function failed_listing_invites(xhr) {
 }
 
 exports.invited_as_values = new Map([
-    [1, i18n.t("Member")],
-    [2, i18n.t("Organization administrator")],
-    [3, i18n.t("Guest")],
-    [4, i18n.t("Organization owner")],
+    [100, i18n.t("Organization owner")],
+    [200, i18n.t("Organization administrator")],
+    [400, i18n.t("Member")],
+    [600, i18n.t("Guest")],
 ]);
```

Comment:
Ideally, we can delete this object and just reuse the values we use for user roles in general.  Should be a follow-up commit though.

Reply:
Using the role values from settings_config.js will require adding a separate function with `if-else` conditions to set the value of `invited_as_text`.
Isn't the Map providing a simple way of setting `invited_as_text` values for invites.

Reply:
We can have `settings_config.js` export a second data structure computed from the first with a dictionary format keyed by the integers, or this tuple format if needed; the point is I don't want us duplicating the data for these mappings; there should be a single source of truth for what these numbers mean.

(Long-term, I expect the `settings_config` data will be sent by the server).